back-end/controllers/UserController.js

```diff
@@ -23,4 +26,24 @@ const updateUserById = async (req, res) => {
   }
 };
 
-module.exports = { getUser, updateUserById };
+const changePassword = async (req, res) => {
+  try {
+    const { currentPassword, newPassword } = req.body;
+    const { id } = req.params;
+    const userPass = await getCurrentPassword(id);
+    const isValidPassword = await bcrypt.compare(currentPassword, userPass.password);
```

Comment:
If we use this method few times in different places in our project, we could create a new service with bcrypt for compare and hash password.

Reply:
created a new service with bcrypt for compare and hash password